web/src/test/java/log4j.xml

```diff
@@ -1,25 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE log4j:configuration SYSTEM "log4j.dtd">
-<log4j:configuration>
```

Comment:
I'm not sure if we can easily remove this. In production it is very important to have async logging as otherwise routing requests could be delayed if disc is busy. This is especially true under high load and when we fetch GBs of new road network data.